elytron/src/main/java/org/wildfly/extension/elytron/CredentialStoreResourceDefinition.java

```diff
@@ -192,7 +216,17 @@ private CredentialStoreAddHandler() {
         protected void performRuntime(OperationContext context, ModelNode operation, Resource resource) throws OperationFailedException {
 
             ModelNode model = resource.getModel();
-            String uri = asStringIfDefined(context, URI, model);
+            String location = asStringIfDefined(context, LOCATION, model);
```

Comment:
Just an FYI, with DMR 1.4.0 the whole asStringIfDefined could go away and you can just use

String location = LOCATION.resolveModelAttribute(context, model).asStringOrNull();

The various ModelNode asXXXOrNull() return null if the node is undefined.

No need to change this; it's just an FYI.